examples/notebooks/beam-ml/run_inference_xgboost.ipynb

```diff
@@ -0,0 +1,321 @@
+        "# to you under the Apache License, Version 2.0 (the\n",
+        "# \"License\"); you may not use this file except in compliance\n",
+        "# with the License. You may obtain a copy of the License at\n",
+        "#\n",
+        "#   http://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing,\n",
+        "# software distributed under the License is distributed on an\n",
+        "# \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+        "# KIND, either express or implied. See the License for the\n",
+        "# specific language governing permissions and limitations\n",
+        "# under the License"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Apache Beam RunInference for XGBoost\n",
+        "\n",
+        "<table align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_xgboost.ipynb\"><img src=\"https://raw.githubusercontent.com/google/or-tools/main/tools/colab_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_xgboost.ipynb\"><img src=\"https://raw.githubusercontent.com/google/or-tools/main/tools/github_32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "</table>\n"
+      ],
+      "metadata": {
+        "id": "DUGbrRuv89CS"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "This notebook demonstrates the use of the RunInference transform for XGBoost. Apache Beam RunInference has implementations of the ModelHandler class prebuilt for XGBoost. For more information about the RunInference API, see Machine Learning in the Apache Beam documentation.\n",
+        "\n",
+        "You can choose the appropriate model handler based on your input data type:\n",
+        "\n",
+        "- NumPy model handler\n",
+        "- Pandas DataFrame model handler\n",
+        "- Datatable model handler\n",
+        "- SciPy model handler\n",
+        "\n",
+        "With RunInference, these model handlers manage batching, vectorization, and prediction optimization for your XGBoost pipeline or model.\n",
+        "\n",
+        "This notebook demonstrates the following common RunInference patterns:\n",
+        "\n",
+        "Generate predictions.\n",
+        "Postprocess results after RunInference.\n",
+        "One model to showcase classification of Iris flowers and one regression model to showcase prediction of housing prices"
+      ],
+      "metadata": {
+        "id": "6nh2h-sIOAOg"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Before you begin\n",
+        "Complete the following setup steps:\n",
```

Comment:
```suggestion
        "Before you begin, complete the following setup steps:\n",
```

Not sure why, but this renders incorrectly in colab and GitHub (the trailing `\n` isn't respected).

<img width="406" alt="image" src="https://user-images.githubusercontent.com/42773683/227998497-36143a83-85f6-4139-9ad0-a7ae1a806b30.png">